notebooks/Access NectarCAM data using DIRAC API.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
<p>Perhaps change "changeme" into something that could be used as vanilla option such as "nectarchain-dirac"? This is not clearly nitpicking, though.</p><p>The only suggestion I could offer would be to have this as an alternate environment yaml file in the main nectarchain directory so that the user can easily install this if they wish to. The notebook then could just say "Install nectarchain using environment-dirac.yml". This could also be left for a future PR though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/nectarchain/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Hi @tibaldo .
Thanks a lot for the suggestion. I agree, it would be much cleaner to offer such an alternative environment file at the root repo level. I'll propose this within this PR.